tests/testdata/certs/default/cert-chain.pem

```diff
@@ -1,20 +1,20 @@
 -----BEGIN CERTIFICATE-----
-MIIDMjCCAhqgAwIBAgIUBR8+06vaC2wEuBS4nO31AV7GsM8wDQYJKoZIhvcNAQEL
-BQAwGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDAgFw0yMjAzMTAyMTU1NTZaGA8y
-Mjk1MTIyNDIxNTU1NlowGDEWMBQGA1UEAwwNY2x1c3Rlci5sb2NhbDCCASIwDQYJ
-KoZIhvcNAQEBBQADggEPADCCAQoCggEBAL7+1gOUdIjbQ8PQFG89AP6jdTT6V4cD
-arIo3FFjkdT20aKgwln2338iuMFKEgnQUjDpHtpgF3Jskv8Wg7eaDP2PbVFmTwos
-zsbzIqIIwRQ43K5maZAvDcC4rXCospQkxIjngpwsNXEujlQnK7mX9HbESl8v6Znt
-55KdqoMy1SFIqc+lVCOjvGaXGnnf9Ju+9M/HkCj604biDYbNQ1oJF45MiaGUdoYa
-WTK0A+LANADYcAAlc0O60agATI+mSmzV9QwJS2vDu3yjyA6M6rIvm52IqV++5Brv
-Z4aI2UHq4y3KDaAPDxOG5sI/HLiZsYEJI2Im8E0Dro54g+AYvxby44ECAwEAAaNy
-MHAwCQYDVR0TBAIwADALBgNVHQ8EBAMCBeAwHQYDVR0lBBYwFAYIKwYBBQUHAwIG
-CCsGAQUFBwMBMDcGA1UdEQQwMC6GLHNwaWZmZTovL2NsdXN0ZXIubG9jYWwvbnMv
-ZGVmYXVsdC9zYS9kZWZhdWx0MA0GCSqGSIb3DQEBCwUAA4IBAQBxtgqEcRO7EcG4
-0uZOs5M2iexvTQVtjFdek8dkDSo401QEgl2Di84cq+JBNTCf7Mjd4FDqqZsyeGEc
-73hlQFGM3WRJXEMqoOu0TmmZI55Bs4hN7FyOMi0y1aPf+O9k+7eiFa5dCz3UKtpK
-cVb46UcYPiA6ODQjVzKRS61evyREV84CCNw/aH3IH9sNI5XJqhc8d5/jACPV2rp9
-kSfpO/eSOqEfWLQF9DYHptVO6RXzSASbWgKp75YXtl2EkdENXVFb4tJbYpO2uLI5
-+UKTykaP3ICQV4fpLe6B57IcxdkSVjVFkxIbpR3PQdwrgz+YMiSAhFh+3PUBYpMM
-7gXilpu4
+MIIDRjCCAi6gAwIBAgIUIf2Vv9QwxTJgEJObGbJvHMthI2YwDQYJKoZIhvcNAQEL
```

Comment:
Is this certificate generated by generate.sh or by yourself?

Reply:
By generate.sh.